README.md

```diff
@@ -37,13 +37,26 @@ Use the SDKs to leverage _[Intelligent Authentication](https://www.forgerock.com
 
 ## Installation
 
+### Cocoapods
+Use following pods in your `Podfile` to install FR iOS SDK module(s) you want to install the latest version.
```

Comment:
Is there a missing "if" in the second half of the sentence:

" ... to install FR iOS SDK module(s) if you want to install the latest version."

Reply:
Maybe even rephrase to "Use the following pods in your `Podfile` in order to install FR iOS SDK module(s)' latest version."